### PR TITLE
Tenant Metadata Route added

### PR DESCRIPTION
**Pull request by pras7430**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-171

 ## Description:
Tenant Metadata Route added.
http://localhost:4200/intelligence/admin/tenanttmetadata
 ## Testing:

Dec-14-2020 21-07-19

 ## Screenshots:
(if applicable)

### CODE REVIEW

File: projects/admin/src/app/tab/tabset-routing.module.ts [hunk 1]
```diff
@@ -38,9 +39,12 @@ const routes: Routes = [
       breadcrumb: 'Event'
     }
   },
-
-
-
+  {
+    path:'tenanttmetadata', component: TenantMetadataListComponent,
```
Comment (lovedeep92) [inline projects/admin/src/app/tab/tabset-routing.module.ts:43]: it should be tenantmetadata!

File: projects/admin/src/app/_service/tenant-meta-data.service 2.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
```
Comment (FarajiA) [inline projects/admin/src/app/_service/tenant-meta-data.service 2.ts:1]: I'm curious as to why the files have `2` in the filename?

File: projects/admin/src/app/tab/tabset-routing.module.ts (hunk 1 above)
Comment (FarajiA) [reply projects/admin/src/app/tab/tabset-routing.module.ts:43]: Correct. I'd also be okay with this path being just `tenant` given that we will be adding create along with this list.

File: projects/admin/src/app/tab/_features/tenantMetadata/pages/tenant-metadata-list/tenant-metadata-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { TenantMetadata} from "../../../../../_model/tenantMetadata";
```
Comment (FarajiA) [inline projects/admin/src/app/tab/_features/tenantMetadata/pages/tenant-metadata-list/tenant-metadata-list.component.ts:2]: This import isn't needed

**Review by FarajiA — APPROVED** [review]

File: projects/admin/src/app/tab/_features/tenantMetadata/pages/tenant-metadata-list/tenant-metadata-list.component.ts (hunk 3 above)
Comment (pras7430) [reply projects/admin/src/app/tab/_features/tenantMetadata/pages/tenant-metadata-list/tenant-metadata-list.component.ts:2]: oh yes, at least not now!

File: projects/admin/src/app/tab/tabset-routing.module.ts (hunk 1 above)
Comment (pras7430) [reply projects/admin/src/app/tab/tabset-routing.module.ts:43]: typo😁.. replace with just "tenant".

File: projects/admin/src/app/_service/tenant-meta-data.service 2.ts (hunk 2 above)
Comment (pras7430) [reply projects/admin/src/app/_service/tenant-meta-data.service 2.ts:1]: 😁😁😁 wrong commit while switching the branch.

**Pull request merged**